src/installer/managed/CommonManaged.props

```diff
@@ -34,20 +25,11 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' != 'netstandard2.0' And '$(TargetFramework)' != 'netcoreapp2.0' ">
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-  </PropertyGroup>
-
   <PropertyGroup>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>git://github.com/dotnet/core-setup</RepositoryUrl>
+    <RepositoryUrl>git://github.com/dotnet/runtime</RepositoryUrl>
```

Comment:
Please remove that property. It's already defined here: https://github.com/dotnet/runtime/blob/b6ef0a54ad0dbe016ec63850e3a70bfacb9bde09/Directory.Build.props#L67.

Reply:
I'm fine if we do that in a follow-up PR.